test/test.jl

```diff
@@ -244,6 +244,20 @@ for val in 10 .^ range(2, stop=-6, length=20)
     sleep(0.1)
 end
 
+# Threshold-based progress reports with increment=false
+println("Testing threshold-based progress")
+prog = ProgressMeter.ProgressThresh(1e-5, "Minimizing:")
+for val in 10 .^ range(2, stop=-6, length=20)
+    ProgressMeter.update!(prog, val; increment=false)
+    sleep(0.1)
+end
```

Comment:
You need to check that it in fact isn't incrementing the counter.

Reply:
Done!